website/common/locales/en/groups.json

```diff
@@ -14,7 +14,7 @@
   "community": "<a target='_blank' href='http://habitica.wikia.com/wiki/Special:Forum'>Community Forum</a>",
   "dataTool": "Data Display Tool",
   "resources": "Resources",
-  "askQuestionNewbiesGuild": "Ask a Question (Habitica Help guild)",
+  "askQuestionNewbiesGuild": "Ask a Question (Newbies Guild)",
```

Comment:
There seem to be a bunch of unrelated edits to this string file, some of them (such as this) clearly incorrect. Mistake resolving a merge conflict?